SpatialGDK/Source/SpatialGDK/Private/Interop/RPCExecutor.cpp

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/RPCExecutor.h"
+
+#include "Interop/Connection/SpatialTraceEventBuilder.h"
+#include "Interop/SpatialPlayerSpawner.h"
+#include "Interop/SpatialReceiver.h"
+#include "Interop/SpatialSender.h"
+#include "Utils/RepLayoutUtils.h"
+
+namespace SpatialGDK
+{
+RPCExecutor::RPCExecutor(class USpatialNetDriver* InNetDriver)
+	: NetDriver(InNetDriver)
+{
+}
+
+bool RPCExecutor::ExecuteCommand(const FCrossServerRPCParams& Params)
+{
+	const TWeakObjectPtr<UObject> TargetObjectWeakPtr = NetDriver->PackageMap->GetObjectFromUnrealObjectRef(Params.ObjectRef);
+	if (!TargetObjectWeakPtr.IsValid())
+	{
+		return false;
+	}
+
+	UObject* TargetObject = TargetObjectWeakPtr.Get();
+	const FClassInfo& ClassInfo = NetDriver->ClassInfoManager->GetOrCreateClassInfoByObject(TargetObjectWeakPtr.Get());
+	UFunction* Function = ClassInfo.RPCs[Params.Payload.Index];
+	if (Function == nullptr)
+	{
+		return true;
+	}
+
+	uint8* Parms = static_cast<uint8*>(FMemory_Alloca(Function->ParmsSize));
```

Comment:
I'm guessing by the use of alloca this is code is copied from somewhere? Does this warrant a comment

Reply:
copied code as well. Can add a comment, but not too familiar with the initial reasoning behind it